app/scripts/controllers/main.js

```diff
@@ -31,12 +26,25 @@ angular.module('scoreTrackerApp')
           $scope.scoreForm.$setPristine();
         }
         $scope.newScore = { name: "", value: undefined };
-
-        updateList();
-        $scope.updateSummary();
-
+        $scope.updateList();
         focusForm();
       }
     };
 
+    $scope.$watch('scores', function() {
```

Comment:
Yes, I'd like to see the logic for summary statistics centralized in the service, and I like the idea of caching them in the service, much more efficient.  I'll take another look at removing the watch in the controller.